homeassistant/components/cover/tahoma.py

```diff
@@ -66,13 +131,56 @@ def device_class(self):
             return 'window'
         return None
 
+    @property
+    def state_attributes(self):
```

Comment:
Do not override `state_attributes`. Devices can override `device_state_attributes`

Reply:
Ok, I'll update it.